extensions/notebook/src/common/utils.ts

```diff
@@ -445,6 +445,7 @@ export function isBookItemPinned(notebookPath: string): boolean {
 export enum BookTreeItemType {
 	Book = 'Book',
 	Notebook = 'Notebook',
+	BookNotebook = 'BookNotebook',
```

Comment:
Why do we need another item type here?

Reply:
BookNotebook is very confusing, what are you trying to refer here PinnedNotebook ?

Reply:
I removed this extra type